### PR TITLE
New SFTP Troubleshooting Step

### LINKED ISSUE #2411: WordPress plugin installation error due to FS_* defines in wp-config.php
Re: https://pantheon.io/docs/more-sftp/#manage-plugins-and-themes-with-wp-admin

_(I'd say put it in troubleshooting section with possibly a link at top of the page in the WP-admin section so its not as easy to miss!)_

### Issue

Document that having `FS_METHOD` and `FS_CHMOD_DIR` / `FS_CHMOD_FILE` lines in *wp-config.php* :

```
define('FS_METHOD', 'direct');
define('FS_CHMOD_DIR', 755);
define('FS_CHMOD_FILE', 644);
```

will prevent installation or updating of plugins with an error like:

> Update Failed: The package could not be installed.

screenshot 2017-04-28 14 47 01

### Solution

Remove these lines from *wp-config.php*.

### Background Info

WPEngine and other hosts include or add these lines and therefore it is common for sites migrating from other providers to have them.

### Goals

We should first add this as an simple help article we can link to in chat (common there!) or tickets, and then as a second phase submit a platform issue to detect/report these problematic defines in *wp-config.php* on the Dashboard and/or status page.

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #2411 

## Effect
PR includes the following changes:
- Implements @ilovelamp's suggested new troubleshooting section.
- Documents a plugin installation issue caused by non-standard `wp-config.php` lines added by other hosts.

## Remaining Work
- [x] Approval from @ilovelamp 
- [ ] Copy Review

### CODE REVIEW

**Comment by ilovelamp** [comment]
Looks good. 👍

File: source/_docs/more-sftp.md [hunk 1]
```diff
@@ -115,3 +115,20 @@ Plugin installed successfully.
...
+On some WordPress sites migrated from other hosting platforms, when installing or updating plugins you may see an error like this:
+
+![Failed Plugin Installation](/source/docs/assets/images/wp-plugin-failed.png)
+
+This is usually caused by the following lines in `wp-config.php`, added by your former host:
+
+```
+define('FS_METHOD', 'direct');
+define('FS_CHMOD_DIR', 755);
+define('FS_CHMOD_FILE', 644);
```
Comment (rachelwhitton) [inline source/_docs/more-sftp.md]: @ilovelamp @alexfornuto would it be better to present this as a permissions conflict and refer to these lines as an example? With a recommendation not to override default file permissions?

**Comment by ilovelamp** [comment]
@alexfornuto - I would leave it exactly above — With the specific `FS_` lines that commonly are the customer's issue.

File: source/_docs/more-sftp.md (hunk 1 above)
Comment (ari-gold) [inline source/_docs/more-sftp.md]: Looks good to document. I've also reached out to BlogVault and asked them to automatically remove these lines from wp-config.php as part of migration. I'll report back if/when that work is complete.

**Pull request merged**